Learn/adapters/LearnApi.swift

```diff
@@ -9,16 +9,34 @@
 import Alamofire
 
 struct LearnApi {
+    static func getProfile(completion: @escaping (Student) -> ()) {
```

Comment:
This is great. Heads up I'm throwing in a ticket to refactor this entire API class.